AddCharacter.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=lower_deck_charaters', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$characterName = $_POST['name'] ?? false;
+$characterSpecies = $_POST['species'] ?? false;
+$characterRank = $_POST['rank'] ?? false;
+$characterImage = $_POST['image'] ?? false;
+
+if (
+    $characterName &&
+    $characterSpecies &&
+    $characterRank &&
+    $characterImage
+) {
+
+    if (strlen($characterName) > 150) {
+        echo "Field exceeds maximum length 150.";
+    } else {
+        return true;
+    }
+
+    if (strlen($characterSpecies) > 150) {
+        echo "Field exceeds maximum length 150.";
+    } else {
+        return true;
+    }
+
+    if (strlen($characterRank) > 150) {
+        echo "Field exceeds maximum length 150.";
+    } else {
+        return true;
+    }
+
+    if (strlen($characterImage) > 10000) {
+        echo "Field exceeds maximum length 10000.";
+    } else {
+        return true;
+    }
+
+    $characterDataQuery = $db->prepare(
+        "INSERT INTO `characters`
+        (`name`,`species`, `rank`, `image`)
+        VALUE (:name, :species, :rank, :image)"
+    );
+
+    $characterDataQuery->bindParam('name', $characterName);
+    $characterDataQuery->bindParam('species', $characterSpecies);
+    $characterDataQuery->bindParam('rank', $characterRank);
+    $characterDataQuery->bindParam('image', $characterImage);
+
+    $characterSaved = $characterDataQuery->execute();
```

Comment:
This query is spot on, now you should move it into a new function in your CharacterData model object